Lib/test/pythoninfo.py

```diff
@@ -571,13 +571,23 @@ def collect_cc(info_add):
     except ImportError:
         args = CC.split()
     args.append('--version')
-    proc = subprocess.Popen(args,
-                            stdout=subprocess.PIPE,
-                            stderr=subprocess.STDOUT,
-                            universal_newlines=True)
-    stdout = proc.communicate()[0]
-    if proc.returncode:
-        # CC --version failed: ignore error
+    proc = None
+    try:
+        proc = subprocess.Popen(args,
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.STDOUT,
+                                universal_newlines=True)
+    except (FileNotFoundError, PermissionError):
```

Comment:
This part is perfectly fine ;-)

Reply:
Hum, honestly, ignoring any OSError would be just fine. There is no need to be specific here.

Reply:
Ok.